pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -920,7 +1053,7 @@ func afwSchedulerEntry(client splcommon.ControllerClient, cr splcommon.MetaObjec
 			}
 
 			// Track the cluster scoped apps to track the bundle push
-			if appSrc.Scope == enterpriseApi.ScopeCluster && deployInfoList[i].DeployStatus != enterpriseApi.DeployStatusComplete {
+			if appSrc.Scope == enterpriseApi.ScopeCluster && appDeployContext.BundlePushStatus.BudlePushStage != enterpriseApi.BundlePushComplete {
```

Comment:
Any reason why we are considering apps that were already installed as well for the tracking?

Reply:
Yeah I will change to have apps not installed yet.